integration_test/agents/agents.go

```diff
@@ -717,20 +717,10 @@ func globForAgentPackage(platform string) (string, error) {
 
 // InstallPackageFromGCS installs the agent package from GCS onto the given Linux VM.
 //
-// gcsPath must point to a GCS Path that contains .deb/.rpm/.goo files to install on the testing VMs. Each agent
-// must have its own subdirectory. For example this would be
-// a valid structure inside AGENT_PACKAGES_IN_GCS when testing metrics & ops-agent:
```

Comment:
The new behavior makes enough sense that I didn't feel that it needed a big comment like this in order to explain it. the new behavior is:

```
AGENT_PACKAGES_IN_GCS
├── ops-agent-google-cloud-1.2.3.deb
├── ops-agent-google-cloud-1.2.3.rpm
└── ops-agent-google-cloud-1.2.3.goo
```